src/util/logging.ts

```diff
@@ -0,0 +1,77 @@
+export enum LogLevel {
+  TRACE = 0,
+  DEBUG = 1,
+  INFO = 2,
+  WARN = 3,
+  ERROR = 4,
+}
+
+/**
+ * Converts the FAUNA_DEBUG environment variable (string) into a LogLevel.
+ *   The intended use is to set FAUNA_DEBUG=0|1.
+ *
+ *   This function will convert null, undefined, empty, or or any string
+ *   starting with "0" to false.
+ *
+ * @param fauna_debug - The String value of FAUNA_DEBUG.
+ */
+export function faunaDebugEnabled(fauna_debug: string | undefined): boolean {
+  if (fauna_debug === undefined || fauna_debug === null) {
+    return false;
+  } else {
+    if (fauna_debug.length == 0) {
+      return false;
+    } else {
+      if (fauna_debug.startsWith("0")) {
```

Comment:
If FAUNA_DEBUG is set to `"0"`, e.g. with `FAUNA_DEBUG=0`, then how does one configure a log level of TRACE?